tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/pom.xml

```diff
@@ -8,6 +8,14 @@
 		<version>1.1.0.BUILD-SNAPSHOT</version>
 	</parent>
 
+	<!--Can remove these specific version when the main project updates netty
+	and reactor dependencies-->
+	<properties>
+		<reactor.version>3.0.2.RELEASE</reactor.version>
```

Comment:
Nothing has changed in the main project. Basically we get a Class Not Found when running the integration test. So that is why the dependencies were added.